assets/settings/default.json

```diff
@@ -612,7 +612,20 @@
     "toolbar": {
       // Whether to display the terminal title in its toolbar.
       "title": true
-    }
+    },
+    // Set the cursor style in the terminal.
```

Comment:
One thing I've noticed is that we do not update existing terminals' cursors when changing the settings?

We can do that, as
https://github.com/zed-industries/zed/blob/f330f094650a36e6f54de6e71750797bd841009c/crates/terminal_view/src/terminal_view.rs#L219 
tracks the settings change and has `self.terminal` inside it, which has `term: Arc<FairMutex<Term<ZedListener>>>,` inside itself, on which we can call `term.set_cursor_style(...)`